src/test/cloudWatchLogs/commands/viewLogStream.test.ts

```diff
@@ -53,14 +54,20 @@ describe('viewLogStreamWizard', async function () {
         const groupName = 'grouper'
         const wizard = new SelectLogStreamWizard(
             new LogGroupNode(region, { logGroupName: groupName }),
-            new MockSelectLogStreamWizardContext([streamName])
+            new MockSelectLogStreamWizardContext([
+                { kind: 'selectedLogStream', region: region, logGroupName: groupName, logStreamName: streamName },
+            ])
         )
         const result = await wizard.run()
 
         assert.ok(result)
-        assert.strictEqual(result?.logGroupName, groupName)
-        assert.strictEqual(result?.logStreamName, streamName)
-        assert.strictEqual(result?.region, region)
+        // HACK: using '===' since following this assertion the exact
+        // type of 'result' is narrowed down to 'selectedLogStream'.
+        // Otherwise we get TS2339 error.
+        assert.ok(result.kind === 'selectedLogStream')
```

Comment:
This is currently the only way to narrow the type without writing an overloaded type for `strictEqual`. We can add one though.